### PR TITLE
Migrate docs into repo

### PR DESCRIPTION
**Pull request by ansermino**
## Summary of changes

Changes introduced in this pull request:

- Copy existing docs repo into `/docs`
- Remove unneeded files (eg. `.github`, `LICENSE`)
- Remove prettier config (only used for one line in the `forest` README)

Note: This PR makes not attempt to reconfigure GHA, dependabot, etc. Additional changes tracked in https://github.com/ChainSafe/forest/issues/5045

## Reference issue to close (if applicable)

Related to https://github.com/ChainSafe/forest/issues/5045

## Change checklist

- [x] I have performed a self-review of my own code,
- [x] I have made corresponding changes to the documentation. All new code adheres to the team's [documentation standards](https://github.com/ChainSafe/forest/wiki/Documentation-practices),
- [x] I have added tests that prove my fix is effective or that my feature works (if possible),
- [x] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

File: docs/.bookignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+README.md
+.github
+.spellcheck.yml
+LICENSE
```
Comment (ansermino) [inline docs/.bookignore:4]: Is this file needed for anything?

File: README.md [hunk 2]
```diff
@@ -321,8 +321,7 @@ Forest is dual licensed under [MIT] + [Apache 2.0].
 [Rust]: https://www.rust-lang.org/
 [Lotus]: https://lotus.filecoin.io/
 [Forest Book]: https://docs.forest.chainsafe.io/
-[Forest Q&A]:
-  https://github.com/ChainSafe/forest/discussions/categories/forest-q-a
+[Forest Q&A]: https://github.com/ChainSafe/forest/discussions/categories/forest-q-a
```
Comment (ansermino) [inline README.md:324]: Removed the line wrapping from prettier

**Review by sudo-shashank — APPROVED** [review]

**Review by lemmih — APPROVED** [review]
